subsys/bluetooth/host/scan.c

```diff
@@ -712,32 +713,36 @@ void bt_hci_le_per_adv_sync_established(struct net_buf *buf)
 	}
 
 	if (!pending_per_adv_sync ||
-	    (!atomic_test_bit(pending_per_adv_sync->flags,
-			      BT_PER_ADV_SYNC_SYNCING_USE_LIST) &&
+	    (!atomic_test_bit(pending_per_adv_sync->flags, BT_PER_ADV_SYNC_SYNCING_USE_LIST) &&
 	     ((pending_per_adv_sync->sid != evt->sid) ||
 	      bt_addr_le_cmp(&pending_per_adv_sync->addr, &evt->adv_addr)))) {
-		struct bt_le_per_adv_sync_term_info term_info;
-
 		BT_ERR("Unexpected per adv sync established event");
+		/* Request terminate of pending periodic advertising in controller */
 		per_adv_sync_terminate(sys_le16_to_cpu(evt->handle));
 
+		unexpected_evt = true;
+	} else {
+		unexpected_evt = false;
+	}
+
+	if ((unexpected_evt == false && evt->status != BT_HCI_ERR_SUCCESS) ||
+	    unexpected_evt == true) {
 		if (pending_per_adv_sync) {
+			struct bt_le_per_adv_sync_term_info term_info;
+
 			/* Terminate the pending PA sync and notify app */
 			term_info.addr = &pending_per_adv_sync->addr;
 			term_info.sid = pending_per_adv_sync->sid;
+			term_info.err = evt->status;
 
 			/* Deleting before callback, so the caller will be able
 			 * to restart sync in the callback.
 			 */
 			per_adv_sync_delete(pending_per_adv_sync);
 
-
-			SYS_SLIST_FOR_EACH_CONTAINER(&pa_sync_cbs,
-						     listener,
-						     node) {
+			SYS_SLIST_FOR_EACH_CONTAINER(&pa_sync_cbs, listener, node) {
 				if (listener->term) {
-					listener->term(pending_per_adv_sync,
-						       &term_info);
+					listener->term(pending_per_adv_sync, &term_info);
```

Comment:
I'm curious as why these lines were changed; I know you are using an auto-formatter, but shouldn't that only auto-format lines you've already changed? 

The number of lines changed in this PR looks to be many more than what is actually needed, which seems unnecessary.

Reply:
That is two-in-one. Auto-formatter changed those lines because I'd have edited them before I created the PR. 

I've updated it to avoid unnecessary changes.